### ISSUE TITLE
Author attribution for terms definition

### ISSUE THREAD

**Issue by FrancescoVit** [FoodOntology/joint-food-ontology-wg#14]
Dear all,

as of our last meeting, we in ONS are in the process of taking up the dietary terms as defined in the shared spreadsheet.

There is one issue I would like to clarify and have feedback on: attribution for terms definition/creation. 
In my opinion, here we have a "forking" in how the WG feels better to act: attribution to the individual researcher or attribution to the work-group?
Damion asked to ORCID if a group ID connected to our single ORCID could be created, but they only create profiles for single researchers or recognized organization. On the other hand they suggested RAID (https://www.raid.org.au/) but I got no response from them yet, and am starting to wonder if the project is still active. I also issued a question on [RG](https://www.researchgate.net/post/How_to_formally_referencing_give_credit_to_an_organization_in_software_ontologies) but got no answers. 

Any of you have some idea or can include someone in this discussion?

We could also use two entries in the annotation, and ROBOT should split them up and include both. One could be ORCID of researcher and the other a reference to the WG, waiting for an ORCID-like identifier for the group. In my opinion, referencing to the WG would be the best option to gain some importance and formality as a whole, but here discussion is open and opinion are obviously welcomed. 

Technically, we have at least two class annotation in IAO we could consider, and they would be:

-term editor (IAO:0000117) 
>Name of editor entering the term in the file. The term editor is a point of contact for information regarding the term. The term editor may be, but is not always, the author of the definition, which may have been worked upon by several people

-definition source (IAO:0000119) 
>Formal citation, e.g. identifier in external database to indicate / attribute source(s) for the definition. Free text indicate / attribute source(s) for the definition. EXAMPLE: Author Name, URI, MeSH Term C04, PUBMED ID, Wiki uri on 31.01.2007 

Any other annotation that we could use? How does other ontologies in the WG behave? In ONS, for example, we simply used IAO:0000117 for all newly defined terms giving attribution to the project.

**Comment by LCCarmody** [comment]
I am sure there are several ways to do it, but I can share how HPO and MAXO (possibly MONDO) do it.

For definitions, we use database_cross_reference (http://www.geneontology.org/formats/oboInOwl#hasDbXref) if the person writes the definition. We also use this annotation when sourcing from somewhere else, like a PubMed article. 

We use created_by (http://www.geneontology.org/formats/oboInOwl#created_by)for anyone creating it.

We have also used contributor (http://purl.org/dc/elements/1.1/contributor)  occasionally in HPO.  

We always use individual ORCID IDs (and not group IDs), but we will add as many as are necessary/requested.  

Not sure if there is a better way, but that is our workflow.

**Comment by ddooley** [comment]
BTW OBI, FoodOn, Genepio use IAO:0000119 'definition source' generally pointing to URL (which could be ORCID) alongside IAO:0000115 definition (not as an annotation to it); and IAO:0000117 'term editor' pointing to ORCID or textual name for editor of term.

If RAID falls through, then perhaps just a URL pointing to the github joint ontology workgroup site would do.

**Comment by FrancescoVit** [comment]
> If RAID falls through, then perhaps just a URL pointing to the github joint ontology workgroup site would do.

I honestly doubt that the RAID initiative is still active. Last activity (twitter and website) seems 2019

**Comment by ddooley** [comment]
Any further findings on this topic, Francesco?

**Comment by FrancescoVit** [comment]
No, never got an answer from the RAID initiative, and never got answers on ResearchGate as well

**Comment by ddooley** [comment]
Well, maybe just offer https://github.com/FoodOntology/joint-food-ontology-wg as a contributor for the more discussed terms?  Leigh's http://purl.org/dc/elements/1.1/contributor ?

**Comment by FrancescoVit** [comment]
> Well, maybe just offer https://github.com/FoodOntology/joint-food-ontology-wg as a contributor for the more discussed terms? Leigh's http://purl.org/dc/elements/1.1/contributor ?

At the moment, I've used the url of the github repository as term editor

**Comment by ddooley** [comment]
That sounds fine for the term contributor/editor.  I'll ask the OBOFoundry operations committee about the preferred relation. I've been using IAO:0000119 definition source if someone made up the definition, or IAO:0000117 term editor if someone was involved in bringing the term in from somewhere. But I've certainly seen dc:contributor used a lot too - and FoodOn uses that to acknowledge authors for FoodOn itself.  Looks like I need to expand that list actually, now that I see it.
